tests/content/test_content_tasks.py

```diff
@@ -0,0 +1,19 @@
+from bulbs.content.models import FeatureType
+from bulbs.content.tasks import update_feature_type_rates
+from bulbs.utils.test import BaseIndexableTestCase
+from bulbs.contributions.models import ContributorRole, FeatureTypeRate
+
+
+class FeatureTypeTasksTestCase(BaseIndexableTestCase):
```

Comment:
Let's test to make sure an arbitrary save on an existing featuretype won't create a duplicate as well